File: api/src/main/java/net/blueshell/api/controller/request/JwtRequest.java
package net.blueshell.api.controller.request;

import lombok.Data;
import lombok.EqualsAndHashCode;
import net.blueshell.api.dto.BaseDTO;

import java.io.Serial;

@EqualsAndHashCode(callSuper = true)
@Data
public class JwtRequest extends BaseDTO {
    @Serial
    private static final long serialVersionUID = 5926468583005150707L;
    private String username;
    private String password;

    public JwtRequest() {
    }

    public JwtRequest(String username, String password) {
        this.username = username;
        this.password = password;
    }
}

File: api/src/main/kotlin/net/blueshell/api/auth/web/dto/request/JwtRequest.kt
package net.blueshell.api.auth.web.dto.request

import io.swagger.v3.oas.annotations.media.Schema
import net.blueshell.api.shared.dto.BaseDTO
import java.io.Serial

@Schema(name = "JwtRequest")
data class JwtRequest(
    var username: String?,
    var password: String?
) : BaseDTO() {
    companion object {
        @Serial
        private const val serialVersionUID = 5926468583005150707L
    }
}
